src/application.ts

```diff
@@ -8,15 +8,21 @@ import {validateApiSpec} from '@loopback/testlab'
 import {MyAuthStrategyProvider} from './providers/auth-strategy';
 import {HelloWorldController} from './controllers/hello-world';
 import {MySequence} from './sequence';
+import {spec} from './spec';
+import {ProductController} from './controllers/product.controller';
+import {ProductRepository} from './repositories/product.repository';
 
 export class HelloWorldApp extends Application {
   constructor() {
     super({
       components: [AuthenticationComponent],
       sequence: MySequence,
     });
+    this.api(spec);
     this.bind(AuthenticationBindings.STRATEGY)
       .toProvider(MyAuthStrategyProvider);
     this.controller(HelloWorldController);
+    this.controller(ProductController);
+    this.bind('repositories.Product').toClass(ProductRepository);
```

Comment:
To be replaced by `this.repository()` when it's merged in.

Reply:
Yes!